_apps/remove-outside-collaborators.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Remove Outside Collaborators
+description: Monitors and instantly removes any outside collaborators added to your Organization.
+slug: remove-outside-collaborators
+screenshots:
+- https://user-images.githubusercontent.com/7875028/44161210-a17c2980-a08a-11e8-8adc-3ef69031d67c.png
+authors: [ hollywood ]
+repository: hollywood/remove-outside-collaborators
+host: https://remove-outside-collaborators.herokuapp.com
+---
+
+# Remove Outside Collaborators
+
+Constant monitoring and instant removal of any Outside Collaborators added to your Organization.
+
+## Features
+
+- Will automatically remove any outside collaborators from an organization.
+- Will create an issue in the repo explaining the action.
+- Monitor only mode will not remove any collaborators but will still create an issue.
+- Can set configuration parameters by using YAML file set in a specific repo for the entire org.
+- Can exclude certain members so that they can be added as outside collaborators.
+- Can set a list of users/groups to cc on every issue created
+
+## Get Started
+
+1. [Configure the App](https://github.com/apps/remove-outside-collaborators)
+2. [Recommended] Create repo named `org-settings` and in it a file `.github/remove-outside-collaborators.yml` as described in the [How it Works](#How-it-Works) section to configure settings (and override defaults)
+3. It will then prevent any outside collaborators (other than any whitelisted members) from being added to your organization.
+
+## How it Works
+
+By default when an Outside Collaborator is added to a Repository, an Issue will be created in the Repository with a warning that a Collaborator was added. [Monitor-Only mode is enabled by default].
+
+A `.github/remove-outside-collaborators.yml` file in the Repository `org-settings` is recommended to override the [default settings](./lib/defaults.js). This Repository will contain global settings for the Organization. You can name the settings Repository anything you'd like, `org-settings` is just the default.
```

Comment:
Instead of `(./lib/defaults.js)` you'll need to perma link to it like: `(https://github.com/Hollywood/remove-outside-collaborators/blob/master/lib/defaults.js)`